src/pages/page404/button/button.css

```diff
@@ -0,0 +1,15 @@
+button {
```

Comment:
we will stye all buttons later and will place it to src/shared or kind of. Then we will be reuse it whenever we need it